web/client/plugins/FeatureEditor.jsx

```diff
@@ -111,7 +111,17 @@ const Dock = connect(createSelector(
   *         }
   *       }]
   *     },
-  *   "editingAllowedRoles": ["ADMIN"]
+  *   "editingAllowedRoles": ["ADMIN"],
+  *   "snapTool": true,
```

Comment:
This should be true by default. Please document in props.

Reply:
So snap tool should be available by default and invisible only if explicitly set to false in localConfig? I've updated code to make it work this way, just asking to be 100% sure that's the case.

Reply:
@tdipisa snappint tool should be always available, also instandard mapstore, isn't it? 
I think it makes easier to test.